webapp/channels/src/components/post_view/channel_intro_message/add_members_button.tsx

```diff
@@ -118,17 +118,12 @@ const MoreThanMaxFreeUsers = ({channel, pluginButtons}: {channel: Channel; plugi
                             className='icon-account-plus-outline'
                             title={formatMessage({id: 'generic_icons.add', defaultMessage: 'Add Icon'})}
                         />
-                        {isPrivate && channel.group_constrained &&
+                        {channel.group_constrained &&
                             <FormattedMessage
                                 id='intro_messages.inviteGropusToChannel.button'
                                 defaultMessage='Add groups'
                             />}
-                        {isPrivate && !channel.group_constrained &&
-                            <FormattedMessage
-                                id='intro_messages.inviteMembersToChannel.button'
-                                defaultMessage='Add people'
-                            />}
-                        {!isPrivate &&
+                        {!channel.group_constrained &&
```

Comment:
Maybe I am missing something, but this logic doesn't seem to be exactly the same as the previous one. Let me put a truth table here:
P = `Add People` message
G = `Add Groups` message

Before:
|          | group | !group |
|----------|-------|--------|
| private  | G     | P      |
| !private | P     | P      |

After:
|          | group | !group |
|----------|-------|--------|
| private  | G     | P      |
| !private | G     | P      |

As you see, if the channel is not private and group constrained it used to show "add people" and not it would say "add groups". Not sure if intended or not.

If not intended, the correct, we would need to update both conditions to be something like:

`isPrivate && channel.group_constrained` and `(!isPrivate || !channel.group_constrained)`

Reply:
@larkox the buttons are now the same whether it's private or not so I removed the `isPrivate` check.